uservice_ccutter/plugins/projecttypes/generic.py

```diff
@@ -3,6 +3,37 @@
 import datetime
 
 
+class NullLogger(object):
```

Comment:
Use the NullHandler and getLogger combo.

https://github.com/kennethreitz/requests/blob/master/requests/__init__.py#L74-L83